src/api/goods.ts

```diff
@@ -6,6 +6,16 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = (goods: Good[]) => {
+  return (
+    goods
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .splice(0, 6)
+  );
+};
 
-export const getRedGoods = () => {};
+export const getRedGoods = (goods: Good[]) => {
+  return (
+    goods.filter(good => good.color === 'red')
+  );
+};
```

Comment:
use getAll function inside get5First and getRedGoods instead of getting goods from arguments